lib/msf/core/reflective_dll_loader.rb

```diff
@@ -52,6 +52,10 @@ def parse_pe(dll)
     pe = Rex::PeParsey::Pe.new(Rex::ImageSource::Memory.new(dll))
     offset = nil
 
+    unless pe and pe.exports
```

Comment:
This was to fix throwing a confusing stacktrace if you passed an invalid dll to inject_dll_data_into_process.

Reply:
This file has since been removed from this PR, and I think in this case having stack traces would probably be useful. If the stack trace is confusing and you have ideas on how to make it better, feel free to open a PR @timwr.